civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -53,6 +53,25 @@
   }
 }
 
+.ui-select-multiple {
+  padding: 3px !important;
+
+  &:after {
+    content: '\f0d7';
+    bottom: 0;
+    font-family: "FontAwesome";
+    display: inline-block;
+    height: 33px;
+    line-height: 35px;
+    position: absolute;
+    right: 0;
+    text-align: center;
+    width: 32px;
+    z-index: 1;
+    -webkit-font-smoothing: antialiased;
```

Comment:
I used `font-smoothing` code from `_custom-select.scss` which was already present, now moved the code to a mixin and added `moz` prefix.
Reference: https://davidwalsh.name/font-smoothing